armsrc/iso14443a.c

```diff
@@ -1284,9 +1273,6 @@ static void TransmitFor14443a(const uint8_t *cmd, uint16_t len, uint32_t *timing
 		LastTimeProxToAirStart = ThisTransferTime;
 	}
 
-	// clear TXRDY
```

Comment:
Why remove this one?

Reply:
This sends a 0x00 byte just to clear TXREADY. In the following loop we are waiting for TXREADY to be set again. This wastes 9.4us.

Reply:
Thanks